acurl/setup.cfg

```diff
@@ -8,16 +8,16 @@ classifiers =
     Intended Audience :: Developers
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
 author = Aaron Ecay
 author_email = aaron.ecay@sky.uk
 maintainer = Sky Identity NFT team
 maintainer_email = matthew.ellis@sky.uk
```

Comment:
I guess we can fix this up while we're here -- do you want to be maintainer of acurl? 😅

Reply:
Do you want leaving as the "author" and just the maintainer updated? Jordan has a branch with an [updated email address](https://github.com/sky-uk/mite/blob/f46729dc3c1439ef8c59395ef8f39b82858c37f8/acurl/setup.cfg#L17).